tests/tests_suite_test.go

```diff
@@ -44,8 +44,12 @@ var (
 )
 
 func getDir() string {
-	_, filename, _, _ := runtime.Caller(1)
-	return path.Dir(filename)
+	if os.Getenv("HOME") != "/root" {
```

Comment:
Definitely open to other suggestions on better ways to determine if we're in a Docker container vs. on a dev's box...